learn/wavemaker-release-notes/v10-7-2.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "WaveMaker 10.7.2 - Release date: 21 June 2021"
+id: ""
+sidebar_label: "Release v10.7.2"
+---
+*WaveMaker releases an update with bug fixes.*
+
+## Improvements
+
+---
+
+- Improved the format of the **Currency** and **Number** widgets to support natural and financial modes by setting the **Input Mode** property.
```

Comment:
Currency and number widgets now support an additional "Financial" mode of entering data. This is very useful in applications dealing with financial data. This mode can be configured by configuring the "Input Mode" property on the widget. By default the value for this property is "natural" mode.